package.json

```diff
@@ -1,4 +1,25 @@
 {
+  "name": "pwr2",
```

Comment:
Do you guys think that pwr2 is enough? ... what about pwr2-cli or pwr2-ui ? ... Just imagine all of the frontend stuff can be packaged with NPM...